frontend/src/metabase/query_builder/components/notebook/lib/steps.types.ts

```diff
@@ -0,0 +1,51 @@
+import type { DatasetQuery } from "metabase-types/types/Card";
+import type StructuredQuery from "metabase-lib/queries/StructuredQuery";
+import type Question from "metabase-lib/Question";
+
+export enum QuestionStepType {
```

Comment:
We don't use enums (because we use babel for compilation), please convert to a `type`, e.g. `export type QuestionStepType = "data" | "join" ...`